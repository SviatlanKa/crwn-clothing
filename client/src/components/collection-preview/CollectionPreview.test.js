import React from 'react';
import { shallow } from 'enzyme';
import CollectionPreview from './CollectionPreview';

it('expect render CollectionPreview component', () => {
    const items = [
        {
            id: 1,
            imageUrl: 'www.images/blue-hat.jpg',
            price: 100,
            name: 'Blue Hat'
        },
        {
            id: 2,
            imageUrl: 'www.images/red-hat.jpg',
            price: 150,
            name: 'Red Hat'
        },
        {
            id: 3,
            imageUrl: 'www.images/white-hat.jpg',
            price: 180,
            name: 'White Hat'
        },
        {
            id: 4,
            imageUrl: 'www.images/green-hat.jpg',
            price: 130,
            name: 'Green Hat'
        }
    ];

    const props = {
        title: "Hats",
        items: items,
        history: "",
        match: "",
        routeName: ""
    };
    const wrapper = shallow(<CollectionPreview.WrappedComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
});