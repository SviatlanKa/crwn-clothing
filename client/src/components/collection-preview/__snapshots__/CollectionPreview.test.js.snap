// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render CollectionPreview component 1`] = `
<div
  className="collection-preview"
>
  <h1
    className="title"
    onClick={[Function]}
  >
    HATS
  </h1>
  <div
    className="preview"
  >
    <Connect(CollectionItem)
      item={
        Object {
          "id": 1,
          "imageUrl": "www.images/blue-hat.jpg",
          "name": "Blue Hat",
          "price": 100,
        }
      }
      key="1"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 2,
          "imageUrl": "www.images/red-hat.jpg",
          "name": "Red Hat",
          "price": 150,
        }
      }
      key="2"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 3,
          "imageUrl": "www.images/white-hat.jpg",
          "name": "White Hat",
          "price": 180,
        }
      }
      key="3"
    />
    <Connect(CollectionItem)
      item={
        Object {
          "id": 4,
          "imageUrl": "www.images/green-hat.jpg",
          "name": "Green Hat",
          "price": 130,
        }
      }
      key="4"
    />
  </div>
</div>
`;
